packages/cli/src/commands/link/__tests__/link-test.js

```diff
@@ -1,7 +1,7 @@
 import {func as link} from '../link';
 import loadConfig from '../../../tools/config';
 import makeHook from '../makeHook';
-
+// $FlowFixMe - converted to TS
```

Comment:
We can migrate the `link-test` and `makeHook-test` to ts as well. We shouldn't need the `FlowFixMe` then

Reply:
I tried migrating but I was getting some weird unrelated errors mostly with namespace... I will try once more then !

Reply:
done !!!